src/components/Guilds/Web3Modals/WalletModal.tsx

```diff
@@ -0,0 +1,272 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { isMobile } from 'react-device-detect';
+import { FiArrowLeft } from 'react-icons/fi';
+import { useWeb3React } from '@web3-react/core';
+
+import { usePrevious } from 'utils';
+import { injected, getWallets, isChainIdSupported } from 'provider/connectors';
+import { useRpcUrls } from 'provider/providerHooks';
+import Option from './components/Option';
+import { Modal } from '../common/Modal';
+import { Heading } from '../common/Typography';
+import WalletInfoBox from './components/WalletInfoBox';
+import Transaction from './components/Transaction';
+import { Button } from '../common/Button';
+
+const Container = styled.div`
+  margin: 2rem;
+`;
+
+const ErrorHeading = styled(Heading)`
+  line-height: 0;
+  margin: 0;
+  margin-bottom: 1.5rem;
+  padding: 0;
+`;
+
+const BackIcon = styled(FiArrowLeft)`
+  height: 1.5rem;
+  width: 1.5rem;
+  cursor: pointer;
+  margin: 0;
+  padding: 0;
+
+  &:hover {
+    color: ${({ theme }) => theme.colors.muted};
+  }
+`;
+
+const Divider = styled.hr`
+  border-top: 1px solid ${({ theme }) => theme.colors.muted};
+`;
+
+const BlockButton = styled(Button)`
+  width: 100%;
+  margin-top: 1.5rem;

```

Comment:
```suggestion
  margin: 1.5rem 0 0;
```
Before: 
![Screen Shot 2021-12-14 at 12 09 15](https://user-images.githubusercontent.com/25844967/146024702-a79af4dc-ac2a-4c4f-8c9b-5f4394c5bccf.png)

After:
![Screen Shot 2021-12-14 at 11 56 03](https://user-images.githubusercontent.com/25844967/146024733-de3eafb6-35de-45bd-afb0-b74b0570dadf.png)